jsonstore/jsonstore.go

```diff
@@ -21,6 +21,10 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+const (
```

Comment:
Can we set it to at least `4`?  so that we can test out the byzantine fault tolerance. Tendermint will continue to produce blocks as long as more than 2/3 of validators are non-malicious.

Reply:
yes you are right, that was originally 4 but I've started decreasing it when I struggled to set up nodes locally.

Reply:
fixed